cdap-examples/Purchase/src/main/java/co/cask/cdap/examples/purchase/PurchaseApp.java

```diff
@@ -69,7 +70,7 @@ public void configure() {
 
     // Schedule the workflow
     schedule(
-      buildSchedule("DailySchedule", "PurchaseHistoryWorkflow")
+      buildSchedule("DailySchedule", SchedulableProgramType.WORKFLOW, "PurchaseHistoryWorkflow")
```

Comment:
I really don't like this class SchedulableProgramType. In the end they are just program types, and the builder validates anyway that it is schedulable. Can we use ProgramType here?